requests/add-openvdb-outputs.yml

```diff
@@ -0,0 +1,4 @@
+action: add_feedstock_output
+feedstock_to_output_mapping:
+  - openvdb: openvdb*
+  - openvdb: nanovdb*
```

Comment:
```suggestion
  - openvdb: openvdb-*
  - openvdb: nanovdb-*
```

Reply:
Thank you for reviewing!

I updated like you suggest, but added ~nanovdb~ without glob as well.

I have used underscore in the output names in the recipe. Would dashes be preferable?

Reply:
Either a dash or underscore is fine. I find dashes aesthetically more please but this is a personal preference.

Reply:
I took a quick look at the referenced example packages with multiple outputs and they all used -, so I'll follow that.

Please merge if you are happy with the req and I'll push with dashes in my PR :).